scripts/get-okteto.sh

```diff
@@ -70,6 +70,20 @@
                 fi
         fi
 
+        download_uri="https://downloads.okteto.com/cli"
+        channel_file="${OKTETO_HOME:-$HOME/.okteto}/channel"
+        
+        if [ -e  $channel_file ];
+        then channel=$(cat $channel_file)
```

Comment:
We probably need more guards around this. What if we can't cat or the contents are something unexpected.